src/dotnet/commands/dotnet-tool/list/LocalizableStrings.resx

```diff
@@ -148,10 +142,16 @@
   <data name="CommandsColumn" xml:space="preserve">
     <value>Commands</value>
   </data>
+  <data name="ManifestFileColumn" xml:space="preserve">
+    <value>Manifest</value>
+  </data>
   <data name="InvalidToolPathOption" xml:space="preserve">
     <value>Tool path '{0}' does not exist.</value>
   </data>
   <data name="LocalOptionDescription" xml:space="preserve">
     <value>List the tools installed in the local tool manifest.</value>
   </data>
+  <data name="ListToolCommandInvalidGlobalAndLocalAndToolPath" xml:space="preserve">
+    <value>The local option(--local), the global option (--global), the tool path option (--tool-path), can only have one at a time. Please specify only one of the options: {0}.</value>
```

Comment:
This text needs review.

cc @KathleenDollard

Reply:
I will double check with her when she is back. although the text is basically the same as _install_ which is reviewed